shared/src/main/scala/mlscript/helpers.scala

```diff
@@ -72,6 +72,7 @@ abstract class TypeImpl extends Located { self: Type =>
     case Literal(IntLit(n)) => n.toString
     case Literal(DecLit(n)) => n.toString
     case Literal(StrLit(s)) => "\"" + s + "\""
+    case Literal(UnitLit(b)) => if(b){"undefined"} else {"null"}
```

Comment:
Formatting is non-standard.

```suggestion
    case Literal(UnitLit(b)) => if (b) "undefined" else "null"
```